WordPress/Classes/ViewRelated/Stats/Referrer Details/ReferrerDetailsCell.swift

```diff
@@ -0,0 +1,67 @@
+import UIKit
+
+final class ReferrerDetailsCell: UITableViewCell {
+    private let referrerLabel = UILabel()
+    private let viewsLabel = UILabel()
+    private let separatorView = UIView()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+// MARK: - Public Methods
+extension ReferrerDetailsCell {
+    func configure(isLast: Bool) {
+        referrerLabel.text = "Test"
+        viewsLabel.text = "123"
+        separatorView.isHidden = !isLast
+    }
+}
+
+// MARK: - Private methods
+private extension ReferrerDetailsCell {
+    func setupViews() {
+        setupReferrerLabel()
+        setupViewsLabel()
+        setupSeparatorView()
+    }
+
+    func setupReferrerLabel() {
+        WPStyleGuide.Stats.configureLabelAsLink(referrerLabel)
+        referrerLabel.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(referrerLabel)
+        NSLayoutConstraint.activate([
+            referrerLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 16),
```

Comment:
Can we move the `16` into a `Constants` struct since it's being used in a few different locations?